internetarchive/openlibrary
# Add OpenStax to trusted book providers

Closes #6071


Adds OpenStax to our trusted book providers.  This means that any read buttons for OpenStax books will direct patrons to the book's page on OpenStax.

### Technical


### Testing

1. Search for books with OpenStax as the publisher. Ensure that search results contain trusted provider read buttons.
2. Click on one of the read buttons. Ensure that  the book's OpenStax details page is opened in a new tab.
3. Click the search results tab.  Ensure that the toast message is present, and that its links work as expected.
4. Navigate to the book page of an OpenStax edition. Ensure that the trusted provider read button is present.
5. Ensure that the download options link works correctly.

### Screenshot


![openstax_book_page](https://user-images.githubusercontent.com/28732543/153514613-f58dbff5-ff58-460a-98b3-6ed6dd6f72c6.png)

_OpenStax read button --- Book page_

![openstax_search_results](https://user-images.githubusercontent.com/28732543/153514695-028531c6-6097-4958-b26b-d954a95c8b3a.png)

_OpenStax read button and toast message --- Search results page_

### Stakeholders

@cdrini
